### PR TITLE
Fix tutorial + R2 mention

### PR DESCRIPTION
**Pull request by paraynaud**

### CODE REVIEW

**Comment by paraynaud** [comment]
@tmigot I reserted my previous change.
I kept the privous neural network and I added a mention to JSOSolvers.R2.

File: tat [hunk 1]
```diff
@@ -0,0 +1,562 @@
+[33mcommit c930d63b6e53a914b7f9b1db762234cdf686262c[m[33m ([m[1;36mHEAD -> [m[1;32mpr-fix-tutorial[m[33m, [m[1;31morigin/pr-fix-tutorial[m[33m)[m
```
Comment (tmigot) [inline tat:1]: I guess we can remove this file?

File: docs/src/tutorial.md [hunk 2]
```diff
@@ -133,3 +122,15 @@ The length of `w` must be `nlp.meta.nvar`.
 g = similar(w)
 NLPModels.grad!(nlp, w, g)
 ```
+
+## Train a neural network with JSOSOlvers.R2
```
Comment (tmigot) [inline docs/src/tutorial.md:126]:
```suggestion
## Train a neural network with JSOSolvers.R2
```

**Review by tmigot — APPROVED** [review]
Thanks @paraynaud ! I think it was important to add.
Checked the rending here https://jso.dev/FluxNLPModels.jl/previews/PR20/ .

I approved, but there are two trivial changes, so feel free to merge after.

File: docs/src/tutorial.md (hunk 2 above)
Comment (paraynaud) [reply docs/src/tutorial.md:126]: Nice catch !!

File: tat (hunk 1 above)
Comment (paraynaud) [reply tat:1]: Do you mean the commit ?

Comment (tmigot) [reply tat:1]: No, I meant this file `tat`?

**Comment by paraynaud** [comment]
>  Checked the rending here https://jso.dev/FluxNLPModels.jl/previews/PR20/ .

It seems correct for me, `R2` trained effectively the neural network.

**Pull request merged**